ui/components/results.jsx

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
 import Suite from './suite.jsx';
+import {Component} from 'react-component-tree';
+
+class Results extends Component {
+
+  get children() {
+    return {
+      suite: (suite, filter, key) => {
+        return {
+          component: Suite,
+          suite: suite,
+          filter: filter,
+          key: key
+        };
+      }
+    };
+  }
 
-class Results extends React.Component {
   render() {
-    const filter = this.props.filter;
+    const filter = this.props.filter,
+          _this = this;
```

Comment:
You can avoid having to hoist `this` here and in other tests by using fat arrows.